<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>1024</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>DHCP Server</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Connected.ico</normaloff>:/Connected.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="configVerticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelInterface">
          <property name="text">
           <string>Interface:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxInterface"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="settingsHorizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutLeft">
          <item>
           <widget class="QLabel" name="labelStartAddress">
            <property name="text">
             <string>First IP address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStartAddress"/>
          </item>
          <item>
           <widget class="QLabel" name="labelSubnetmask">
            <property name="text">
             <string>Subnet mask:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSubnetmask">
            <property name="text">
             <string>255.255.255.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutRight">
          <item>
           <widget class="QLabel" name="labelEndAddress">
            <property name="text">
             <string>Last IP address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditEndAddress"/>
          </item>
          <item>
           <widget class="QLabel" name="labelLeasetime">
            <property name="text">
             <string>Lease time (s):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxLeasetime">
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>86400</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonRestoreDefaultSettings">
        <property name="text">
         <string>Restore default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSaveSettings">
        <property name="text">
         <string>Save settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonServerState">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start server</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="assignmentsVerticalLayout">
      <item>
       <widget class="QLabel" name="labelDynamicAssignments">
        <property name="text">
         <string>Dynamic assignments:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="dynamicAssignmentsTableView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>110</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>21</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelStaticAssignments">
        <property name="text">
         <string>Static assignments:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="staticAssignmentsTableView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>110</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>21</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowDebugOutput">
        <property name="text">
         <string>Show debug output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="debugOutputText">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
